### ISSUE TITLE
Test case makedhcp_remote_network failed in travis node

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5480]
Test case ``makedhcp_remote_network`` failed in travis test environment.
But can be successful in our own test environment.

```
   '------START::makedhcp_remote_network::Time:Mon Jul 30 02:52:05 2018------',
          '',
          'RUN:mkdef -t network -o testnetwork net=100.100.100.0 mask=255.255.255.0 mgtifname=\'!remote!eth0\' gateway=100.100.100.1 [Mon Jul 30 02:52:05 2018]',
          'ElapsedTime:1 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          '1 object definitions have been created or modified.',
          'CHECK:rc == 0	[Pass]',
          ' ',
          'RUN:lsdef -t network [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          '172_17_0_0-255_255_0_0  (network)',
          '192_168_122_0-255_255_255_0  (network)',
          'testnetwork  (network)',
          'CHECK:rc == 0	[Pass]',
          'CHECK:output =~ testnetwork	[Pass]',
          'CHECK:rc == 0	[Pass]',
          ' ',
          'RUN:mkdef -t node -o testnode ip=100.100.100.2 groups=all mac=42:3d:0a:05:27:0b [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          '1 object definitions have been created or modified.',
          'CHECK:rc == 0	[Pass]',
          ' ',
          'RUN:cp -f /etc/hosts /etc/hosts.bak [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          'RUN:echo "100.100.100.2 testnode" >> /etc/hosts [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          'CHECK:rc == 0	[Pass]',
          ' ',
          'RUN:makedhcp -d testnode [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:1 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          'RUN:makedhcp testnode [Mon Jul 30 02:52:07 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          'CHECK:rc == 0	[Pass]',
          ' ',
          'RUN:makedhcp -q testnode [Mon Jul 30 02:52:07 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          'CHECK:rc == 0	[Pass]',
          'CHECK:output =~ testnode: ip-address = 100.100.100.2	[Failed]',
          ' ',
          'RUN:makedhcp -d testnode [Mon Jul 30 02:52:07 2018]',
          'ElapsedTime:1 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          'RUN:makedhcp -n [Mon Jul 30 02:52:08 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          'Warning: [travis-job-175c6840-cc44-4439-b06d-b9b3668fddeb]: No dynamic range specified for 10.20.0.1. If hardware discovery is being used, a dynamic range is required.',
          'Warning: [travis-job-175c6840-cc44-4439-b06d-b9b3668fddeb]: No dynamic range specified for 172.17.0.0. If hardware discovery is being used, a dynamic range is required.',
          'Warning: [travis-job-175c6840-cc44-4439-b06d-b9b3668fddeb]: No dynamic range specified for 192.168.122.0. If hardware discovery is being used, a dynamic range is required.',
          'Renamed existing dhcp configuration file to  /etc/dhcp/dhcpd.conf.xcatbak',
          '',
          ' ',
          'RUN:noderm testnode [Mon Jul 30 02:52:08 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          'RUN:chtab -d netname=testnetwork networks [Mon Jul 30 02:52:08 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          'RUN:cp -f /etc/hosts.bak /etc/hosts [Mon Jul 30 02:52:08 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          ' ',
          '------END::makedhcp_remote_network::Failed::Time:Mon Jul 30 02:52:08 2018 ::Duration::3 sec------',
```

**Comment by hu-weihua** [comment]
related task https://github.com/xcat2/xcat2-task-management/issues/255

**Comment by hu-weihua** [comment]
After debug, below lines did work as we expected,
```
'RUN:echo "100.100.100.2 testnode" >> /etc/hosts [Mon Jul 30 02:52:06 2018]',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          'CHECK:rc == 0	[Pass]',
```
In travis, the running result is 
```
RUN:cat /etc/hosts',
          'ElapsedTime:0 sec',
          'RETURN rc = 0',
          'OUTPUT:',
          '',
          '127.0.1.1 travis-job-2d358181-8474-4f71-a4cb-fa048bbadd35 travis-job-2d358181-8474-4f71-a4cb-fa048bbadd35 ip4-loopback trusty64',
          '',
          '127.0.0.1 localhost nettuno travis vagrant 100.100.100.2 testnode ',
```
That result in the rest operation failed.

**Comment by hu-weihua** [comment]
This issue will be fixed by PR https://github.com/xcat2/xcat-core/pull/5418

**Comment by hu-weihua** [comment]
Has been fixed by #5418. close this issue
